syngenta_digital_dta/s3/adapter.py

```diff
@@ -55,12 +55,24 @@ def put(self, **kwargs):
             ACL=acl,
             Body=body,
             Bucket=self.bucket,
-            Key=kwargs['s3_path']
+            Key=kwargs['s3_path'],
+            Tagging=kwargs.get('tags', '')
```

Comment:
we need both utilities to accept tags as a `dict()` which contains key-value pairs for the tags, then the utilities modify construct the tag string from that dict and pass it to the boto3 util